index.html

```diff
@@ -202,6 +202,7 @@
                 .setPopup(new mapboxgl.Popup().setMaxWidth('250px').setHTML(markerHtml))
                 .addTo(map);
 
+              const urgentNeed = item.gsx$urgentneed.$t ? `<h3 style="color: #f00; font-size: 80%">Urgent Need: ${item.gsx$urgentneed.$t}</h3>` : ''
```

Comment:
```suggestion
              const urgentNeed = location.urgentNeed ? `<h3 style="color: #f00; font-size: 80%">Urgent Need: ${location.urgentNeed}</h3>` : ''
```

Reply:
Sorry, this stuff is pretty messy. I've been writing it since 10:30pm last night.